<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSender</class>
 <widget class="QWidget" name="DataSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Sender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="boxInput">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="rbBinary">
              <property name="text">
               <string>&amp;Binary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbText">
              <property name="text">
               <string>&amp;Text</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="editData">
            <property name="placeholderText">
             <string>Data to Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbCodec"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSend">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxAction">
         <property name="title">
          <string>Action</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="cmbHandler"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="textLog"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
